pkg/detectors/aws/aws_test.go

```diff
@@ -333,8 +345,13 @@ func TestAWS_FromChunk(t *testing.T) {
 					t.Fatalf("wantVerificationError %v, verification error = %v", tt.wantVerificationError, got[i].VerificationError())
 				}
 			}
-			ignoreOpts := cmpopts.IgnoreFields(detectors.Result{}, "RawV2", "Raw", "verificationError")
-			if diff := cmp.Diff(got, tt.want, ignoreOpts); diff != "" {
+			ignoreOpts := []cmp.Option{
+				cmpopts.IgnoreFields(detectors.Result{}, "RawV2", "Raw", "verificationError"),
+				cmpopts.SortSlices(func(x, y detectors.Result) bool {
+					return x.Redacted < y.Redacted
+				}),
```

Comment:
Why is this necessary now?

Reply:
This test case was flakey because it sometimes arrived in a different order:

```found two, returned both because the active secret for one paired with the inactive ID, despite the hash```